### PR TITLE
Feature/11664 onboarding docs

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Task: [AB#11664](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11664)
Activity: [AB#11264](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11264)

### CODE REVIEW

File: docs/local-k8s-setup.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
+# Installation
+
 Depending on your machine setup please choose the most appropriate Installation Method from below
 
+Please the relevant option:
```
Comment (bjss-chris) [inline docs/local-k8s-setup.md:5]: Should this be 'Please see...'?

File: docs/rancher-setup.md [hunk 2]
```diff
@@ -1,17 +1,64 @@
 # Rancher Setup
 
-## Setup instructions modification
-When using Rancher, the recommended certificate option is to use Rancher generated certificates. However, there is a modification to the step in the provided [instructions](https://rancher.com/docs/rancher/v2.x/en/installation/k8s-install/helm-rancher/). 
+## Add the Helm Chart Repository
+Use helm repo add command to add the Helm chart repository that contains charts to install Rancher. For more information about the repository choices and which is best for your use case, see Choosing a Version of Rancher.
```
Comment (bjss-chris) [inline docs/rancher-setup.md:4]: 'Use *the* `helm repo add` command'? (And suggest marking helm repo add as code).

Should 'Choosing a Version of Rancher' be a link?

File: docs/run-local.md [hunk 3]
```diff
@@ -27,6 +35,8 @@ helm install bc ingress-nginx/ingress-nginx
 
 ### Add Namespace
 
+If this is not initial build, please ensure you are pointing at your local Kubernetes desktop version.
```
Comment (bjss-chris) [inline docs/run-local.md:38]: 'If this is not *the* initial build'?

**Review by bjss-chris — CHANGES_REQUESTED** [review]
A couple of suggestions. There are also linting issues in all three files – recommend adding the `markdownlint` extension in VS Code.

(I wonder if there's a way of running linting on .md files automatically? Something for later...)

File: docs/run-local.md (hunk 3 above)
Comment (PhysicsJon) [reply docs/run-local.md]: Updated

File: docs/rancher-setup.md (hunk 2 above)
Comment (PhysicsJon) [reply docs/rancher-setup.md]: Updated

File: docs/local-k8s-setup.md (hunk 1 above)
Comment (PhysicsJon) [reply docs/local-k8s-setup.md]: Updated

**Review by bjss-chris — DISMISSED** [review]

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**